src/interface/src/app/plan/create-scenarios/set-priorities/set-priorities.component.spec.ts

```diff
@@ -14,6 +14,10 @@ import { PlanService } from './../../../services/plan.service';
 import { ConditionsConfig } from './../../../types/data.types';
 import { ColormapConfig } from './../../../types/legend.types';
 import { SetPrioritiesComponent } from './set-priorities.component';
+import { Component } from '@angular/core';
+
+@Component({ selector: 'app-scenario-tooltip', template: '' })
+class ScenarioTooltipMockComponent {}
```

Comment:
This is the _nice_ way of getting rid of the not a known element. Other quicker option  is to use `CUSTOM_ELEMENTS_SCHEMA` but mostly to hide the warning.

Other option is to actually include the real component on the test, however I think that's the worst bc you'll also need to inject all the dependencies and your unit tests becomes a mess